KsApi/models/GraphUserCreditCard.swift

```diff
@@ -27,6 +27,11 @@ public struct GraphUserCreditCard: Swift.Decodable {
     }
   }
 
+  public enum PaymentType: String, Swift.Decodable, CaseIterable {
+    case applePay = "APPLE_PAY"
+    case card = "CREDIT_CARD"
```

Comment:
I wonder if `creditCard` would be more explicit here so we can differentiate between credit cards and debit cards in the future.